extension/popup.js

```diff
@@ -101,6 +101,114 @@ const downloadStatusIndicator = document.getElementById(
   "downloadStatusIndicator"
 );
 
+
+// DOM elements (theme)
+const lightTheme = document.getElementById("lightTheme");
+const darkTheme = document.getElementById("darkTheme");
+const container = document.querySelector(".container");
+const tagLine = document.querySelector(".tagline");
+const tags = document.querySelectorAll("h3");
+const shortcutInfo = document.querySelector(".shortcut-info");
+const codeTags = document.querySelectorAll("code");
+const smallTag = document.querySelector("small");
+const footer = document.querySelector(".footer");
+const bodyTag = document.querySelector("body");
+const githubLink = document.querySelector(".github-link a");
+
+// theme toggle
+lightTheme.addEventListener("click", () => {
+  lightTheme.style.display = "none";
+  darkTheme.style.display = "flex";
+  container.style.backgroundColor = "white";
+  container.style.color = "black";
+  tagLine.style.color = "var(--dark-gray)";
+  settingsToggleBtn.style.backgroundColor = "#313131";
+  tags.forEach((tag) => (tag.style.color = "var(--dark-gray)"));
+  settingsToggleBtn.style.backgroundColor = "#f1f3f4";
+  settingsToggleBtn.style.color = "black";
+  shortcutInfo.style.backgroundColor = "#f1f3f4";
+  shortcutInfo.style.color = "black";
+  shortcutInfo.style.borderColor = "#f1f3f4";
+  shortcutInfo.style.borderStyle = "solid";
+  shortcutInfo.style.borderWidth = "1px";
+  codeTags.forEach((tag) => {
+    tag.style.backgroundColor = "#f1f3f4";
+    tag.style.color = "black";
+  });
+  smallTag.style.color = "#5f6368";
+  previewContainer.style.backgroundColor = "#f1f3f4";
+  previewContainer.style.color = "black";
+  previewContainer.style.borderWidth = "1px";
+  previewContainer.style.borderStyle = "soild";
+  previewContent.style.color = "black";
+  previewContent.style.backgroundColor = "white";
+  previewContent.style.borderColor = "#f1f3f4";
+  previewContent.style.borderWidth = "1px";
+  previewContent.style.borderStyle = "solid";
+  footer.style.backgroundColor = "white";
+  footer.style.color = "#5f6368";
+  bodyTag.style.backgroundColor = "white";
+  githubLink.style.backgroundColor = "#f1f3f4";
+  githubLink.style.color = "#5f6368";
+  githubLink.style.borderColor = "#f1f3f4";
+  githubLink.style.borderStyle = "solid";
+  githubLink.style.borderWidth = "1px";
+  githubLink.addEventListener("mouseenter", () => {
+    githubLink.style.color = "#4285f4";
+  });
+  githubLink.addEventListener("mouseleave", () => {
+    githubLink.style.color = "#5f6368";
+  });
```

Comment:
Why are we adding mouseenter and mouseleave event listeners to githubLink every time the theme is toggled? This can cause multiple listeners to accumulate, leading to bugs.

**Suggested Fix:**
Add these listeners once when initializing, not inside the theme toggle handler.